sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClientsOptions.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Azure.Security.ConfidentialLedger
 {
-    /// <summary> Client options for ConfidentialLedgerClient. </summary>
-    public partial class ConfidentialLedgerClientOptions : ClientOptions
+    /// <summary> Client options for ConfidentialLedger library clients. </summary>
+    public partial class ConfidentialLedgerClientsOptions : ClientOptions
```

Comment:
Why the change to plural Clients?

Reply:
Because there are multiple clients.

Reply:
Per the guidelines: "If the options type can be shared by multiple client types, name it with a plural or more general name. For example, the BlobClientsOptions class can be used by BlobClient, BlobContainerClient, and BlobAccountClient."
See: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-usage-options

Reply:
Makes sense, thanks!
![](https://media3.giphy.com/media/d9BcF6jtV2R5VXDALq/giphy.gif)